src/cur.jl

```diff
@@ -65,7 +65,8 @@ end
 
 This function retrieves key/data pairs from the database.
 """
-function get{T}(cur::Cursor, key, ::Type{T}, op::CursorOps=FIRST)
+get{K,T}(cur::Cursor, key::K, ::Type{T}, op::CursorOps=FIRST) = get(cur, [key], T)
+function get{K<:Union{String,Array},T}(cur::Cursor, key::K, ::Type{T}, op::CursorOps=FIRST)
```

Comment:
Why put restriction on a key type?

Reply:
There is no restriction. Just that the some key types need to be wrapped in an `Array` to be passed as a pointer (in `MDBValue`) to `ccall`.

Reply:
As I wrote in comment to `src/common.jl`, database interface should be invariant to key/value type.